packages/web-components/src/slider/slider.template.ts

```diff
@@ -1,46 +1,29 @@
 import { html, ref } from '@microsoft/fast-element';
 import type { ElementViewTemplate } from '@microsoft/fast-element';
 import { staticallyCompose } from '../utils/template-helpers.js';
-import { SliderOptions } from './slider.options.js';
-import { Slider } from './slider.js';
+import type { SliderOptions } from './slider.options.js';
+import type { Slider } from './slider.js';
 
 export function sliderTemplate<T extends Slider>(options: SliderOptions = {}): ElementViewTemplate<T> {
   return html<T>`
     <template
-      role="slider"
       tabindex="${x => (x.disabled ? null : 0)}"
-      aria-valuetext="${x => x.valueTextFormatter(x.value)}"
-      aria-valuenow="${x => x.value}"
-      aria-valuemin="${x => x.min}"
-      aria-valuemax="${x => x.max}"
-      aria-disabled="${x => (x.disabled ? true : void 0)}"
-      aria-readonly="${x => (x.readOnly ? true : void 0)}"
-      aria-orientation="${x => x.orientation}"
-      class="${x => x.orientation}"
+      @pointerdown="${(x, c) => x.handlePointerDown(c.event as PointerEvent)}"
     >
-      <div part="positioning-region" class="positioning-region">
-        <div ${ref('track')} part="track-container" class="track">
-          <slot name="track"></slot>
-          <div part="track-start" class="track-start" style="${x => x.position}">
-            <slot name="track-start"></slot>
-          </div>
-        </div>
-        <slot></slot>
-        <div
-          ${ref('thumb')}
-          part="thumb-container"
-          class="thumb-container"
-          style="${x => x.position}"
-          @mousedown="${(x, c) => x.handleThumbMouseDown(c.event as MouseEvent)}"
-          @touchstart="${(x, c) => x.handleThumbMouseDown(c.event as MouseEvent)}"
-        >
-          <slot name="thumb">${staticallyCompose(options.thumb)}</slot>
-        </div>
+      <div ${ref('track')} part="track-container" class="track" style="${x => x.position}"></div>
```

Comment:
Seems like we can set the CSS variables in code directly to the stylesheet so we can avoid having inline styles here and below.

Reply:
Thanks! Updated to using `this.$fastController.addStyles()` and `this.$fastController.removeStyles()`.

Reply:
@marchbox @mlijanto can we run this a few times to ensure that this isn't going to cause perf issues? Normally I would be open to using `addStyles()`, but consider how often and rapidly that's going to be called - is it more performant to update the stylesheet or just pass the value to the template? This change actually has me nervous without digging in.

Reply:
Yeah I was just thinking about that as I pushed the commit. I'll do some perf tests.

Reply:
I'm leaning towards avoiding on first iteration until we can really stress test the API. There's a not-insignificant amount of JS that I think is going to get executed and updated throughout that which may impact memory/CPU?

Reply:
Sounds good. I'll revert it and add this to my TODO list.